CMake build error due to incorrect absolute RN_DIR path

I'm getting a cmake error when building the react-native application with react-native-v8 version 1.4.0

FAILURE: Build failed with an exception.

```
* What went wrong:
Execution failed for task ':react-native-v8:generateJsonModelDebug'.
> C:\DEV\...\node_modules\react-native-v8\android\CMakeLists.txt : C/C++ debug|armeabi-v7a : CMake Error at C:\DEV\p...\node_modules\react-native-v8\android\CMakeLists.txt:117 (add_library):
    Syntax error in cmake code when parsing string

      C:\DEV\...\node_modules\react-native/ReactCommon/jsiexecutor/jsireact/JSIExecutor.cpp;C:\DEV\projects\delijn\ControleApp\node_modules\react-native/ReactCommon/jsiexecutor/jsireact/JSINativeModules.cpp

    Invalid character escape '\D'.
```

I believe it's because following piece of code returns the RN_DIR path using backward slashes (while it should be forward slashes?):
https://github.com/Kudo/react-native-v8/blob/11b0a6de66bcde8e39dd6b7cdd3b2b625b018776/android/build.gradle#L14-L23

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -10,6 +10,7 @@
 
 import groovy.json.JsonSlurper
 import org.apache.tools.ant.filters.ReplaceTokens
+import org.apache.tools.ant.taskdefs.condition.Os
 
 
 File findNodePackageDir(String packageName, boolean absolute = true) {
@@ -21,6 +22,14 @@ File findNodePackageDir(String packageName, boolean absolute = true) {
   }
   def dir = new File(proc.text.trim()).getParentFile()
   return absolute ? dir.getAbsoluteFile() : dir
+}
+
+String toPlatformFileString(File path) {
+  def result = path.toString()
+  if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+    result = result.replace(File.separatorChar, '/' as char)
+  }
+  return result
 }
 
 def safeExtGet(prop, fallback) {
@@ -157,7 +166,7 @@ android {
         arguments "-DANDROID_STL=c++_shared",
                   "-DBOOST_VERSION=${BOOST_VERSION}",
                   "-DBUILD_DIR=${buildDir}",
-                  "-DRN_DIR=${reactNativeDir}",
+                  "-DRN_DIR=${toPlatformFileString(reactNativeDir)}",
                   "-DREACT_NATIVE_TARGET_VERSION=${rnMinorVersion}",
                   "-DV8_ANDROID_DIR=${v8AndroidDir}",
                   "-DSO_DIR=${extractSoDir}"

